src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -127,9 +121,17 @@ public override async Task<bool> Execute()
                 {
                     exactId = await client.FindPackageId(this.Id);
                 }
-                catch (Octokit.NotFoundException)
+                catch (Exception e)
                 {
-                    Logger.ErrorLocalized(nameof(Resources.RepositoryNotFound_Error), this.WingetRepoOwner, this.WingetRepo);
+                    if (e is Octokit.NotFoundException)
+                    {
+                        Logger.ErrorLocalized(nameof(Resources.RepositoryNotFound_Error), this.WingetRepoOwner, this.WingetRepo);
+                    }
+                    else if (e is Octokit.RateLimitExceededException)

```

Comment:
Why rate limit check here, but not elsewhere, like in NewCommand.cs?

Reply:
Moved rate limit checking to the LoadGitHubClient method.